packages/host/app/services/matrix-service.ts

```diff
@@ -627,6 +624,37 @@ export default class MatrixService extends Service {
     } as CardMessageContent);
   }
 
+  public async addSkillCardsToRoom(
+    roomId: string,
+    skillCards: SkillCard[],
+  ): Promise<void> {
+    let attachedSkillEventIds = await this.getCardEventIds(
+      skillCards,
+      roomId,
+      this.skillCardHashes,
+      { includeComputeds: true, maybeRelativeURL: null },
+    );
+    let skillEventIdsStateEvent: Record<string, any> = {};
+    try {
+      skillEventIdsStateEvent = await this.client.getStateEvent(
+        roomId,
+        SKILLS_STATE_EVENT_TYPE,
+        '',
+      );
+    } catch (e) {
+      // this is fine, it just means the state event doesn't exist yet
```

Comment:
Could have checked "instanceof MatrixError" but I think that would result in importing the whole matrix sdk that I think we've been trying to avoid. Instead I checked the errcode for the relevant error message.